bitex/interface/bitstamp.py

```diff
@@ -79,9 +79,8 @@ def _place_order(self, pair, price, size, side, market=None, **kwargs):
         """Place an order with the given parameters."""
         payload = {'amount': size, 'price': price}
         payload.update(kwargs)
-        if market:
-            return self.request('%s/market/%s/' % (side, pair), authenticate=True, params=payload)
-        return self.request('%s/%s/' % (side, pair), authenticate=True, params=payload)
+        suffix = 'market/' if market else ''
+        return self.request('%s/%s%s/' % (side, suffix, pair), authenticate=True, params=payload)
```

Comment:
Why is it the preferred way if even you don't use it all the time? ;)

I see this notation is used in most of the interface files, that's why I kept it like that... so I'd say for the sake of consistency to leave it like that... eventually a new contributor will add new interfaces and he can use the .str() notation there. At least it should be consistent in the same file. Fair enough?

Things like this can be even noted in the contributing.md file, which I think I'll update and submit a PR for it.